src/views/LeadAdd/index.js

```diff
@@ -398,7 +398,7 @@ class LeadAdd extends React.PureComponent {
                     // NOTE: hard-coded confidentiality value
                     confidentiality: 'unprotected',
                     // NOTE: hard-coded priority value
-                    priority: 'low',
+                    priority: 100,
```

Comment:
It would be even better if we get this from leadOptions.priority[0]
Setting this here would be problematic.
Or, just not fill it altogether

Reply:
Fixed